Funcky.Xunit/Funcky.Xunit.csproj

```diff
@@ -23,6 +23,7 @@
     </PropertyGroup>
     <ItemGroup>
         <PackageReference Include="xunit.assert" Version="2.4.0" />
+        <PackageReference Include="xunit.core" Version="2.4.0" />
```

Comment:
You should reference `xunit.extensibility.core`. See: [What NuGet Packages Should I Use?](https://xunit.net/docs/nuget-packages)

Reply:
> Seems like the test want's to execute our library because I added a dependency on xunit.core ...
This should be fixed too when referencing the correct xunit library.